src/components/ContentSidebar/ActivityFeed/version/VersionError.js

```diff
@@ -1,32 +0,0 @@
-/**
- * @flow
- * @file Version Error component
- */
-
-import * as React from 'react';
-import { FormattedMessage } from 'react-intl';
-
-import messages from '../../../messages';
-
-import './Version.scss';
-
-function getErrorMessage(errorCode: string): React.Node {
```

Comment:
This component was specifically to display the 'tooManyVersions' error which no longer gets returned from the API